### DISCUSSION TITLE
multiple @model_validators break a model

### DISCUSSION THREAD

**Discussion by MrSamuelLaw** [pydantic/pydantic#8314] [Question]
First time asking a question here, so please be patient with me.

I have a class and with multiple @model_validator(mode='after') wrapped functions in my class the model breaks.
What I mean by that is for some reason for all model_validators after the first self is None. I'm not sure if this is a bug or intended behavior. I put an example below for clarity.

```python
class Foo(BaseModel):
    x1: int
    x2: int

    @model_validator(mode='after')
    def validate_aspect_one(self):
        pass # this first one works

    @model_validator(mode='after')
    def validate_aspect_two(self):
        pass # this one breaks, self will be None instead of instance of Foo
```

**Comment by Viicos** [comment]
You need to `return self` for after model validators (https://github.com/pydantic/pydantic/pull/8319)

**Reply by MrSamuelLaw** [reply]
Thank you, I missed that in the docs.
